src/components/forms/ChallengeGoalForm.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import {
+  Text,
+  View,
+  StyleSheet,
+  TextInput,
+  TouchableOpacity,
+  Image,
+} from "react-native";
+
+import Stack from "../ui/Stack";
```

Comment:
let's try to use relative paths when possible, in the future if we have to move files around this will be helpful as the relative path won't change

import Stack from "@src/components/ui/Stack";

Reply:
Done